environment.yml

```diff
@@ -17,3 +17,4 @@ dependencies:
 - requests
 - openpyxl
 - geopandas
+- pyarrow
```

Comment:
Was there a reason to specifically add `pyarrow`? (it's certainly useful, but I don't know if we for example read a parquet file anywhere?)

Reply:
It was required to run the hvplot/datashader code in the viz-landscape notebook